eng/versioning/version_client.txt

```diff
@@ -53,7 +53,7 @@ com.azure:azure-analytics-synapse-managedprivateendpoints;1.0.0-beta.5;1.0.0-bet
 com.azure:azure-analytics-synapse-monitoring;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-chat;1.0.1;1.1.0
 com.azure:azure-communication-callingserver;1.0.0-beta.4;1.0.0-beta.5
-com.azure:azure-communication-common;1.0.4;1.0.4
+com.azure:azure-communication-common;1.0.3;1.0.4
```

Comment:
You'd also need to change callingserver pom to update the referenced version to get the pipeline to pass

Reply:
Weird, then I don't understand how anyone was able to check this in since 1.0.4 isn't released yet so 'dependency' should cause build failures:

<version>1.0.4</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->

Reply:
Now it's released, I've updated the file